packages/frontend/src/components/Explorer/CustomDimensionModal/index.tsx

```diff
@@ -1,431 +1,40 @@
-import {
-    BinType,
-    fieldId,
-    isCustomDimension,
-    isDimension,
-    snakeCaseName,
-    type BinRange,
-} from '@lightdash/common';
-import {
-    ActionIcon,
-    Button,
-    Flex,
-    Group,
-    Modal,
-    NumberInput,
-    Radio,
-    Stack,
-    Text,
-    TextInput,
-    Title,
-} from '@mantine/core';
-import { useForm } from '@mantine/form';
-import { IconX } from '@tabler/icons-react';
-import { useEffect, useMemo } from 'react';
-import useToaster from '../../../hooks/toaster/useToaster';
+import { isCustomBinDimension, isDimension } from '@lightdash/common';
```

Comment:
Moved this file content to `packages/frontend/src/components/Explorer/CustomDimensionModal/CustomBinDimensionModel.tsx` 

This component only decides what custom dimension component to render.